src/satellite_images_nso/api/nso_georegion.py

```diff
@@ -48,12 +73,15 @@ class nso_georegion:
 
     def __init__(
         self,
-        path_to_geojson: str,
         output_folder: str,
         username: str,
         password: str,
+        path_to_geojson: str = None,
+        coordinates: str = None,
```

Comment:
I'm confused, I thought coordinates was a list of coordinates for a polygon, not a string?